packages/grid/_modules_/grid/hooks/utils/useGridApiRef.ts

```diff
@@ -22,7 +24,10 @@ export function useGridApiRef(...args): any {
     apiRef.current = {
       unstable_eventManager: new EventManager(),
       state: {} as GridState,
+      instanceId: globalId,
```

Comment:
We could use `unstable_useId` from @mui/utils`

Reply:
I tried. `unstable_useId` generates the id asynchronously. I need it as soon as possible so even the first selector called is called in its instance cache.